acceptance-tests/src/main/java/hudson/plugin/gradle/BuildScansInjectionSettings.java

```diff
@@ -43,6 +43,18 @@ public void setGradleEnterpriseServerUrl(URI server) {
         setBuildScansInjectionFormValue(SERVER_FIELD, server.toString());
     }
 
+    public void clickEnforceUrl() {
+        ensureConfigPage();
+
+        control(by.checkbox("Enforce Gradle Enterprise server url")).click();
+    }
+
+    public void clickAllowUntrustedServer() {
+        ensureConfigPage();
+
+        control(by.checkbox("Allow untrusted server")).click();
```

Comment:
We use these 2 methods together quite often now, worth extracting the method. 

No need to do it in this PR can be done later.

Reply:
done in this PR